.github/GOVERNANCE.md

```diff
@@ -0,0 +1,82 @@
+# Roles and responsibilities
+
+Kindly is maintained through volunteer work - all contributions are welcome and we look to invite new collaborators at all times.
+The roles described below are intended as guidelines for different ways to get involved and different levels of commitment you may choose to take on at any point.
+
+How much you contribute and how long for is entirely at your discretion and collaborators can change their roles in the community in any way at any time that is relevant to them.
+
+There are two simple steps to taking on any role:
+
+- 🔨 **Do** the work involved.  
+  This community is run on willful collaboration, meaning we invite and expect everyone to identify the areas of Kindly they would like to help with and take any reasonable action towards improving the platform.  
+  If you are interested in being a maintainer, find ways in which you can help maintainers with their tasks;  
+  If you see an opportunity to improve the performance of our code, publish an issue suggesting a spike or changes
+
+- 🙋 **Ask** for any help needed.  
+  The codebase on Kindly is the result of a large community collaborating over time, so if there's something you're not sure about it's very likely that someone else will have an insight that can help.  
+  Be active in our discord server, share updates in your issues and if you would like to contribute in different ways let the core team know about your ideas
+
+## Collaborator
+
+Everyone involved in maintaining / improving Kindly is considered a collaborator. All you need in order to be a collaborator is the willpower to become involved.
```

Comment:
maybe replace "All you need in order to be a collaborator is the willpower to become involved." 

with "All you need to become a collaborator is to get involved!"

Reply:
yep, definitely